src/main/java/core/basesyntax/handler/impl/ReturnTransactionHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.TransactionHandler;
+import java.util.Map;
+
+public class ReturnTransactionHandler implements TransactionHandler {
+    private static final int DEFAULT_VALUE = 0;
+
+    @Override
+    public void handleTransaction(Map<String, Integer> mapFruitQuantity, String key, int value) {
+        mapFruitQuantity.put(key, mapFruitQuantity.getOrDefault(key, DEFAULT_VALUE) + value);
```

Comment:
same as supply
you can extract this logic to some Dao layer instead of passing map directly, try to wrap it into something